src/map/utils/battleutils.cpp

```diff
@@ -1207,7 +1207,7 @@ namespace battleutils
 
             uint8 enspell = (uint8)PAttacker->getMod(Mod::ENSPELL);
 
-            if (enspell == ENSPELL_BLOOD_WEAPON)
+            if (enspell == ENSPELL_BLOOD_WEAPON && PDefender->m_EcoSystem != ECOSYSTEM::UNDEAD)
```

Comment:
I wonder how this got forgotten the last time this section was re-written..

Reply:
Actually shouldn't it just drain for zero in the log but still animate? its been a long time since I've seen anyone mistakenly use it vs undead on retail, so I'm not 100% sure.

Reply:
> Actually shouldn't it just drain for zero in the log but still animate? its been a long time since I've seen anyone mistakenly use it vs undead on retail, so I'm not 100% sure.

Yeah, that I am not sure about that either.

Reply:
I guess I'll go check in a few

Reply:
ok, really doesn't animate!